ui/pages/settings/security-tab/__snapshots__/security-tab.test.js.snap

```diff
@@ -34,6 +34,7 @@ exports[`Security Tab should match snapshot 1`] = `
     <div
       class="settings-page__security-tab-sub-header"
     >
+      <span />
```

Comment:
this is expected. Added 
```
<span ref={this.settingsRefs[16]} />
```

to ui/pages/settings/security-tab/security-tab.component.js for a graceful search landing, rather than cutting off "Security alerts"  with its descriptions, when searching "Blockaid" or "OpenSea"